src/Runner.php

```diff
@@ -133,6 +138,11 @@ private function configureContainer(ContainerInterface $container)
         $container->add(UserSiteMembership::class);
         $container->add(UserOrganizationMemberships::class);
         $container->add(UserOrganizationMembership::class);
+        $container->add(OrganizationSiteMemberships::class);
+        $container->add(OrganizationSiteMembership::class);
+        $container->add(OrganizationUserMemberships::class);
+        $container->add(OrganizationUserMembership::class);
+        $container->add(Organization::class);
```

Comment:
Yeah. And we're still only about halfway done. We may want to think about doing some sort of auto-discovery. But I think this is fine for now.